coordinator/gscoordinator/kubernetes_launcher.py

```diff
@@ -635,28 +801,40 @@ def _dump_resource_object(self):
             resource[self._coordinator_service_name] = "Service"
         self._resource_object.dump(extra_resource=resource)
 
-    def create_analytical_instance(self):
-        if not (self._with_analytical or self._with_analytical_java):
+    def _allocate_analytical_engine(self):
+        # check the engine flag
+        if self._with_analytical and self._with_analytical_java:
```

Comment:
Don't raise errors. technically, analytical is a subset of analytical-java, so just go ahead with analytical-java.